src/cluster/master.go

```diff
@@ -30,10 +32,18 @@ func (c *Cluster) AddMicrocontroller(newMC mc.Config) (response PeerUpdateMessag
 	var newGuy mc.Microcontroller
 	newGuy.Load(newMC)
 	newGuy.ID = c.generateUniqueID()
+	if viper.GetString("ENV") == "production" {
+		for _, micro := range c.Microcontrollers {
```

Comment:
this is the guard against 2 copies of gofire running on 1 microcontroller

Reply:
This is fine but for the record I feel like this is overkill. You would have to pretty consciously start multiple instances on the same MC